pkg/device_plugin/server.go

```diff
@@ -21,7 +21,7 @@ const (
 	// ResourceNameSGX is resource name registered to kubelet.
 	ResourceNameSGX = vendor + "/sgx_epc_MiB"
 
-	serverSock             = devicepluginapi.DevicePluginPath + "/sgx.sock"
+	ServerSock             = devicepluginapi.DevicePluginPath + "sgx.sock"
```

Comment:
Using Path.Join(devicepluginapi.DevicePluginPath + "sgx.sock") may be better whether devicepluginapi.DevicePluginPath is endwith / or not.